Corona-Warn-App/src/main/assets/privacy_de.html

```diff
@@ -25,28 +133,25 @@
     <strong>6. Wofür werden Ihre Daten verarbeitet?</strong>
 </p>
 <p>
-    <strong>7. Wie funktioniert das länderübergreifende Warnsystem?</strong>
+    <strong>7. Welche Berechtigungen benötigt die App?</strong>
 </p>
 <p>
-    <strong>8. Welche Berechtigungen benötigt die App?</strong>
+    <strong>8. Wann werden Ihre Daten gelöscht?</strong>
 </p>
 <p>
-    <strong>9. Wann werden Ihre Daten gelöscht?</strong>
+    <strong>9. An wen werden Ihre Daten weitergegeben?</strong>
 </p>
 <p>
-    <strong>10. An wen werden Ihre Daten weitergegeben?</strong>
+    <strong>10. Werden Ihre Daten in Länder außerhalb der EU übermittelt?</strong>
 </p>
 <p>
-    <strong>11. Werden Ihre Daten in Länder außerhalb der EU übermittelt?</strong>
+    <strong>11. Wie können Sie Ihr Einverständnis zurücknehmen?</strong>
 </p>
 <p>
-    <strong>12. Wie können Sie Ihr Einverständnis zurücknehmen?</strong>
+    <strong>12. Welche weiteren Datenschutzrechte haben Sie?</strong>
 </p>
 <p>
-    <strong>13. Welche weiteren Datenschutzrechte haben Sie?</strong>
-</p>
-<p>
-    <strong>14. Datenschutzbeauftragte und Kontakt</strong>
```

Comment:
The word docs say there are 14 points, but number 7 got deleted. Only 13 points remain.